shell_integration/windows/OCUtil/RegistryUtil.cpp

```diff
@@ -12,10 +12,10 @@
  * details.
  */
 
-#include "stdafx.h"
-
 #include "RegistryUtil.h"
 
+#include <Windows.h>
```

Comment:
Nitpick: for consistency's sake you could make this lowercase

Reply:
Uh juck thx